<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestLedMainDialog</class>
 <widget class="QDialog" name="QtTestLedMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestLed</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="widget_led" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="ribi::QtLedWidget" name="led_left" native="true"/>
      </item>
      <item>
       <widget class="ribi::QtLedWidget" name="led_mid" native="true"/>
      </item>
      <item>
       <widget class="ribi::QtLedWidget" name="led_right" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QSlider" name="slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QWidget" name="widget_text" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="text_left">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="text_mid">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="text_right">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ribi::QtLedWidget</class>
   <extends>QWidget</extends>
   <header>qtledwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
